apps/01PigLatin.js

```diff
@@ -4,14 +4,65 @@ var assert = require('assert');
 var prompt = require('prompt');
 prompt.start();
 
+var word = 'CRaz';
+var vowelIndex = -1;
 
 function pigLatin(word) {
-
     // Your code here
 
+    if ( ( word.indexOf('a') > -1 && word.indexOf('a') < vowelIndex ) || vowelIndex === -1 ) {
+        vowelIndex = word.indexOf('a');
+    }
+    if ( ( word.indexOf('A') > -1 && word.indexOf('A') < vowelIndex ) || vowelIndex === -1 ) {
+        vowelIndex = word.indexOf('A');
+    }
+    if ( ( word.indexOf('e') > -1 && word.indexOf('e') < vowelIndex ) || vowelIndex === -1 ) {
+        vowelIndex = word.indexOf('e');
+    }
+    if ( ( word.indexOf('E') > -1 && word.indexOf('E') < vowelIndex ) || vowelIndex === -1 ) {
+        vowelIndex = word.indexOf('E');
+    }
+    if ( ( word.indexOf('i') > -1 && word.indexOf('i') < vowelIndex ) || vowelIndex === -1 ) {
+        vowelIndex = word.indexOf('i');
+    }
+    if ( ( word.indexOf('I') > -1 && word.indexOf('I') < vowelIndex ) || vowelIndex === -1 ) {
+        vowelIndex = word.indexOf('I');
+    }
+    if ( ( word.indexOf('o') > -1 && word.indexOf('o') < vowelIndex ) || vowelIndex === -1 ) {
+        vowelIndex = word.indexOf('o');
+    }
+    if ( ( word.indexOf('O') > -1 && word.indexOf('O') < vowelIndex ) || vowelIndex === -1 ) {
+        vowelIndex = word.indexOf('O');
+    }
+    if ( ( word.indexOf('u') > -1 && word.indexOf('u') < vowelIndex ) || vowelIndex === -1 ) {
+        vowelIndex = word.indexOf('u');
+    }
+    if ( ( word.indexOf('U') > -1 && word.indexOf('U') < vowelIndex ) || vowelIndex === -1 ) {
+        vowelIndex = word.indexOf('U');
+    }
+    if ( ( word.indexOf('y') > -1 && word.indexOf('y') < vowelIndex ) || vowelIndex === -1 ) {
+        vowelIndex = word.indexOf('y');
+    }
+    if ( ( word.indexOf('Y') > -1 && word.indexOf('Y') < vowelIndex ) || vowelIndex === -1 ) {
+        vowelIndex = word.indexOf('Y');
+    }
+
+    if (vowelIndex >0){
```

Comment:
Also, I'm not seeing [Step 3.1](https://github.com/AustinCodingAcademy/intermediate-textbook/blob/master/app-guides/01PigLatin.md#step-31) covered in here.

This is the case where there is a vowel at the beginning of the word.